### PR TITLE
update versions

### PR DESCRIPTION
**Pull request by cdietrich**
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
please review

File: xtend-website/_config.yml [hunk 1]
```diff
@@ -36,10 +36,10 @@ kramdown:
     coderay_css:               style
 
 src:
-  xtext: "https://github.com/eclipse/xtext/blob/v2.7.0"
+  xtext: "https://github.com/eclipse/xtext/blob/v2.11.0"
 
 javadoc:
-  xtext: "http://download.eclipse.org/modeling/tmf/xtext/javadoc/2.8"
+  xtext: "http://download.eclipse.org/modeling/tmf/xtext/javadoc/2.11"
```
Comment (dhuebner) [inline xtend-website/_config.yml]: There is only 2.9 javadoc available  ATM

Comment (cdietrich) [inline xtend-website/_config.yml]: yes but this is targeting the 2.11 release? will it not be available then?

Comment (dhuebner) [inline xtend-website/_config.yml]: It was just a note.
We need a checklist for 2.11 release and add "Publish Javadoc 2.11" point to it.

**Comment by spoenemann** [comment]
I fixed all code links in the documentation and updated the config accordingly, so this change is obsolete.

**Pull request closed without merging**